src/store/slices/node/actionsAsync.ts

```diff
@@ -559,122 +821,216 @@ const getPrometheusMetricsThunk = createAsyncThunk(
   },
 );
 
+// Helper action to update the isFetching state
+const setAliasesFetching = createAction<boolean>('node/setAliasesFetching');
+const setInfoFetching = createAction<boolean>('node/setInfoFetching');
+const setMetricsFetching = createAction<boolean>('node/setMetricsFetching');
+const setAddressesFetching = createAction<boolean>('node/setAddressesFetching');
+const setBalancesFetching = createAction<boolean>('node/setBalancesFetching');
+const setChannelsFetching = createAction<boolean>('node/setChannelsFetching');
+const setPeersFetching = createAction<boolean>('node/setPeersFetching');
+const setPeerInfoFetching = createAction<boolean>('node/setPeerInfoFetching');
+const setEntryNodesFetching = createAction<boolean>('node/setEntryNodesFetching');
+const setSettingsFetching = createAction<boolean>('node/setSettingsFetching');
+const setStatisticsFetching = createAction<boolean>('node/setStatisticsFetching');
+const setTicketsFetching = createAction<boolean>('node/setTicketsFetching');
+const setTokensFetching = createAction<boolean>('node/setTokensFetching');
+const setVersionFetching = createAction<boolean>('node/setVersionFetching');
+const setTransactionsFetching = createAction<boolean>('node/setTransactionsFetching');
+
```

Comment:
this can maybe move somewhere else, seems to add noise and complexity here. We can tackle this in a different pr but i would say that this is not ideal

Reply:
create an issue to make sure we don't forget this.

Reply:
Issue made